src/uuid/mod.rs

```diff
@@ -436,40 +474,46 @@ impl Binary {
     }
 }
 
-#[cfg(feature = "uuid-0_8")]
-#[cfg_attr(docsrs, doc(cfg(feature = "uuid-0_8")))]
-impl From<uuid::Uuid> for Binary {
-    fn from(uuid: uuid::Uuid) -> Self {
-        Binary {
-            subtype: BinarySubtype::Uuid,
-            bytes: uuid.as_bytes().to_vec(),
+macro_rules! trait_impls {
+    ($feat:meta, $u:ty) => {
+        #[$feat]
+        #[cfg_attr(docsrs, doc($feat))]
+        impl From<$u> for Binary {
+            fn from(uuid: $u) -> Self {
+                Binary {
+                    subtype: BinarySubtype::Uuid,
+                    bytes: uuid.as_bytes().to_vec(),
+                }
+            }
         }
-    }
-}
 
-#[cfg(all(feature = "uuid-0_8", feature = "serde_with"))]
-#[cfg_attr(docsrs, doc(cfg(all(feature = "uuid-0_8", feature = "serde_with"))))]
-impl<'de> serde_with::DeserializeAs<'de, uuid::Uuid> for crate::Uuid {
-    fn deserialize_as<D>(deserializer: D) -> std::result::Result<uuid::Uuid, D::Error>
-    where
-        D: serde::Deserializer<'de>,
-    {
-        let uuid = Uuid::deserialize(deserializer)?;
-        Ok(uuid.into())
-    }
-}
+        #[$feat]
```

Comment:
these ones need to also only be implemented when `"serde_with"` is enabled

Reply:
Good catch, fixed.